model/zos/resources.yaml

```diff
@@ -0,0 +1,74 @@
+groups:
+  - id: resource.zos.software
```

Comment:
Is there any special meaning or flavor `software` adds? Why not just `zos` ?

Reply:
Comment from mainframe SIG: this is meant to refer to software that is running on the zos system itself. We anticipate that there might be other subclasses of resource.zos beyond this. Is this distinction consistent with the rest of the OTel semantics? If not, then we'll remove the software subclass.